orderbook/src/database/orders.rs

```diff
@@ -78,6 +78,20 @@ impl Database {
             .map(|_| ())
     }
 
+    pub async fn cancel_order(&self, order: &Order) -> Result<()> {
+        const QUERY: &str = "\
+            UPDATE orders
+            SET cancellation_timestamp = $1 \
```

Comment:
We should probably only set it if its not already set? Otherwise I could mess with the history by re-cancelling an already cancelled order.

Reply:
In this PR, since we are going to go  with the simple boolean for now, overwriting true with true shouldn't be a problem. However, I will create an issue about using timestamps that mentions this as well.